cmd/juju/commands/bootstrap.go

```diff
@@ -332,12 +334,12 @@ var (
 	waitForAgentInitialisation = common.WaitForAgentInitialisation
 )
 
-var ambiguousDetectedCredentialError = errors.New(`
+var errorAmbiguousDetectedCredential = errors.New(`
```

Comment:
We don't want these renames.
ambiguousDetectedCredentialError is ok
ambiguousCredentialError is also ok